### DISCUSSION TITLE
PHOTO RAW with Max Lens 2.0

### DISCUSSION THREAD

**Discussion by euromarco** [gopro/labs#1047] [Feature Requests]
ciao ragazzi,
on Gopro 12 black, you can create a code to take raw photos with MLM 2.0 ?
Thank you, Marco

**Comment by dnewman-gpsw** [comment]
Not possible within MLM modes. Raw photo implies no image processing and the storage of the "raw" CFA Bayer pixel data.  Turn off MLM and you can take Raw, but you will see the full image circle. This is what raw has to be. So leave MLM attached, with menu set to MLM off.

**Reply by euromarco** [reply]
Grazie
